docs/api_autopilot.md

```diff
@@ -0,0 +1,56 @@
+# API autopilot
+
+Kusk gateway supports API auto deployment. 
+
+Any REST API will typically consist out of a Kubernetes pod that is running the server and a Kubernetes service pointed to it.
+
+## It just works
+To expose preexisting REST API in the cluster you would just have to execute this:
+
+```sh
+kubectl apply -f svc.yaml
+```
+Or edit existing Kubernetes service to add annotation `kusk-gateway/openapi-url` with URL to the location of your OpenAPI
+
+## Under the hood
+
+Let's explain what is going on.
+
+We added a convenience method that will allow users to easily expose their REST API through Kusk gateway by using `kusk-gateway/openapi-url` annotation, and here is how.
+
+Assuming that the user has already set up a pod that is running REST API server code and the pod name is `todo-backend`:
```

Comment:
Generally they will set up a Deployment?

Reply:
I was thinking about the same. I thought here I would go to the lowest possible level which is pods. If you think it is confusing then I don't mind changing it

Reply:
Please do. The normal flow is to expose a deployment which manages the pods, rather than the pods directly

Reply:
```suggestion
Assuming that the user has already set up a pod that is running REST API server code and the deployment name is `todo-backend`:
```
Like this?